### ISSUE TITLE
more operative systems ? 

### ISSUE THREAD

**Issue by anibalardid** [iduspara/codeimg_feedback#3]
Hi ! 
Your app is amazing  !

Could you add more operative systems (linux distros) ? Like Ubuntu, Elementary, etc ? 

Kinds

**Comment by igor-duspara** [comment]
Yeah, we are planing to add new window themes. We want to cover the default window style for each OS, for more specific design we will add the feature to let the user upload custom icons.
